src/main/java/com/clozex/shop/service/impl/BookServiceImpl.java

```diff
@@ -31,10 +31,11 @@ public BookDto save(CreateBookRequestDto requestDto) {
     }
 
     @Override
-    public List<BookDto> findAll() {
-        return bookRepository.findAll().stream()
+    public Page<BookDto> findAll(Pageable pageable) {
+        List<BookDto> bookDtos = bookRepository.findAll(pageable).stream()
                 .map(bookMapper::toDto)
                 .toList();
+        return new PageImpl<>(bookDtos);
```

Comment:
The Page has map method 

```suggestion
        return bookRepository.findAll(pageable)
            .map(bookMapper::toDto);
```